build.cake

```diff
@@ -108,28 +107,35 @@ Task("Generate-Coverage")
 .IsDependentOn("Build")
 .Does(() => 
 {
-	try
+	if(IsRunningOnWindows())
 	{
-		OpenCover(
-			tool => 
-			{
-				tool.DotNetCoreTest("./tests/Test.Xive/",
-				new DotNetCoreTestSettings
+		try
+		{
+			OpenCover(
+				tool => 
 				{
-					 Configuration = "Release"
-				});
-			},
-			new FilePath("./coverage.xml"),
-			new OpenCoverSettings()
-			{
-				OldStyle = true
-			}
-			.WithFilter("+[Xive]*")
-		);
+					tool.DotNetCoreTest("./tests/Test.Xive/",
+					new DotNetCoreTestSettings
+					{
+						Configuration = "Release"
+					});
+				},
+				new FilePath("./coverage.xml"),
+				new OpenCoverSettings()
+				{
+					OldStyle = true
+				}
+				.WithFilter("+[Xive]*")
+			);
+		}
+		catch(Exception ex)
+		{
+			Information("Error: " + ex.ToString());
+		}
 	}
-	catch(Exception ex)
+	else
 	{
-		Information("Error: " + ex.ToString());
+Verbose("Is Linux no  coverage");
```

Comment:
Please use indentation
Please use a correct phrasing (like "Generating coverage is not supported on Linux")